prebid_cache_client/client.go

```diff
@@ -129,7 +130,13 @@ func encodeValueToBuffer(value Cacheable, leadingComma bool, buffer *bytes.Buffe
 
 	buffer.WriteString(`{"type":"`)
 	buffer.WriteString(string(value.Type))
-	buffer.WriteString(`","value":`)
+	if value.TTLSeconds > 0 {
+		buffer.WriteString(`","ttlseconds":`)
+		buffer.WriteString(string(value.TTLSeconds))
```

Comment:
This doesn't do what you think it does... see: https://play.golang.org/p/khS3hM1y7v9

Needs to be `strconv.Itoa(value.TTLSeconds)`. Please add a test to catch this.